code/game/machinery/computer/supply.dm

```diff
@@ -14,6 +14,7 @@
 	var/active_category = null
 	var/menu_tab = 0
 	var/list/expanded_packs = list()
+	authorization = 0
```

Comment:
Isn't this already declared on Line 10?

Reply:
So it is, I completely overlooked it because I saw the var/ and just erased it from my mind. Fixed now.